README.md

```diff
@@ -1,6 +1,12 @@
 # Appeal Planning Decision API
 
-## Commands
+## Setup
+
+cd api-microservice
```

Comment:
Probably worth putting all these inside backticks to style it up as code - would be worth doing everywhere, even on the things already in there

Reply:
👍